zkPySyft/core/zokrates.py

```diff
@@ -56,6 +78,9 @@ def setup(self):
     def compute_witness(self):
         # TODO: read args from file
         args = " ".join(self.input_assignments)
+        if self.gadgets:
```

Comment:
Since `self.gadgets` is initialized as an empty list, I think we should drop the check and go directly to the `for` loop